server/src/main/java/io/druid/server/coordination/BatchDataSegmentAnnouncer.java

```diff
@@ -84,7 +84,7 @@ public void announceSegment(DataSegment segment) throws IOException
     synchronized (lock) {
       // create new batch
       if (availableZNodes.isEmpty()) {
-        SegmentZNode availableZNode = new SegmentZNode(makeServedSegmentPath(new DateTime().toString()));
+        SegmentZNode availableZNode = new SegmentZNode(makeServedSegmentPath());
```

Comment:
maybe we can generalize RealtimeIndexTask.makeTaskId for this instead of UUIDs?
It makes it easier to debug when paths have an actual timestamp.